app/src/main/res/values-en/strings.xml

```diff
@@ -113,34 +113,34 @@ The info you enter is then directly added to the OpenStreetMap in your name, wit
     <string name="short_no_answer_on_button">No</string>
 
     <string name="quest_leave_new_note_title">"Leave a note instead?"</string>
-    <string name="quest_leave_new_note_description">"You can leave a public note at this location which other mappers can comment on and hopefully resolve or alternatively just hide the quest for yourself only"</string>
-    <string name="quest_leave_new_note_yes">"OK"</string>
-    <string name="quest_leave_new_note_no">"No, just hide"</string>
+    <string name="quest_leave_new_note_description">"You can leave a public note for other mappers to resolve at this location, or stop seeing this quest"</string>
+    <string name="quest_leave_new_note_yes">"Leave a note"</string>
+    <string name="quest_leave_new_note_no">"Hide this quest"</string>
 
     <string name="quest_generic_otherAnswers">"Other answers…"</string>
     <!-- aka "not applicable", "not answerable" -->
     <string name="quest_generic_answer_notApplicable">"Can’t say…"</string>
 
     <string name="quest_generic_answer_does_not_exist">It does not exist…</string>
-    <string name="osm_element_gone_description">"Are you sure that it does not exist, not even at a slightly different location? When in doubt, better leave a note instead.</string>
+    <string name="osm_element_gone_description">"Are you sure that it does not exist, not even at a slightly different location? If you're not sure, leave a note instead.</string>
     <string name="osm_element_gone_confirmation">It does not exist</string>
-    <string name="leave_note">Leave note</string>
+    <string name="leave_note">Leave a note</string>
 
     <string name="quest_generic_answer_differs_along_the_way">"Differs along the way…"</string>
-    <string name="quest_split_way_description">If it differs along the way, the first step is to split up the way. After that, the quest can be answered for each split part separately.\nSplit it now?</string>
-    <string name="quest_split_way_tutorial">Tap on the way to split it at the point(s) where the answer differs. Try to be as precise as possible, you can zoom in as usual.</string>
+    <string name="quest_split_way_description">If it differs along the way, the first step is to split up the way. After that, the quest can be answered for each part separately.\nSplit it now?</string>
+    <string name="quest_split_way_tutorial">Tap on the way to split it at the point(s) where the answer differs. Use the zoom function and try to be as precise as possible.</string>
     <string name="quest_split_way_too_imprecise">"Please zoom in further"</string>
     <string name="quest_split_way_many_splits_confirmation_description">These are quite a few splits. You can always further split up the way later.</string>
 
-    <string name="quest_source_dialog_title">Are you sure you checked this on-site?</string>
+    <string name="quest_source_dialog_title">Are you sure you personally observed this?</string>
```

Comment:
I'm not personally convinced this is better.

Reply:
Restored